notebooks/metrics_demo.ipynb

```diff
@@ -122,9 +135,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "cuml_kmeans = KMeans(n_clusters=10)\n",
-    "X_blobs = StandardScaler().fit_transform(X_blobs)\n",
-    "cu_y_pred = cuml_kmeans.fit_predict(X_blobs).to_array()\n",
+    "from cuml import DBSCAN as cumlDBSCAN\n",
```

Comment:
It would be nice to have some visuals of the resulting models. 

For example, it would be nice to provide a matplotlib chart for the clustering demo that showcases the distribution of the test dataset, perhaps with the datapoints colored by their resulting cluster. 

For the classification demo, it would be nice to display the confusion matrix. Scikit-learn has a great example of how to do this. 

For the regression demo, we might be able to display a chart w/ the data and the resulting regression line. We might be able to do this in 3d as well.

The dimensionality reduction demo will chart nicely.

Reply:
Simon has a PR out for cuml to return confusion matrix, i can use that to create the confusion matrix and also showcase the new feature once the PR is merged:
https://github.com/rapidsai/cuml/pull/1817